Sources/GRPCCodeGen/CodeGenerationRequest.swift

```diff
@@ -84,7 +84,13 @@ public struct CodeGenerationRequest {
   public struct Dependency: Equatable {
     /// If the dependency is an item, the property's value is the item representation.
     /// If the dependency is a module, this property is nil.
-    public var item: Item? = nil
+    public var item: Item?
+
+    /// The access level to be included in imports of this dependency.
+    ///
+    /// - Note: This defaults to `public`, which is the default access level for imports on compilers
+    /// up to and including Swift 6.0.
```

Comment:
Bit misleading because we don't default it. I think you mean if it isn't explicitly set then assumed implicit default is public.

Reply:
Hm yes but with this API this isn't an optional argument, we're not defaulting it so it must always be explicit. I'll just get rid of this comment (unless we want to default it in the `init`)